project/dexter-core/src/java/com/samsung/sec/dexter/core/config/PeerReviewConfigFile.java

```diff
@@ -0,0 +1,38 @@
+package com.samsung.sec.dexter.core.config;
+
+import java.util.Map;
+
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+import com.samsung.sec.dexter.core.util.FileService;
+
+public class PeerReviewConfigFile extends DexterConfigFile {
+	private FileService fileService;
+	
+	public PeerReviewConfigFile(FileService fileService) {
+		this.fileService = fileService;
+	}
+	
+	protected void checkDexterConfigMap(final Map<String, Object> map) {
+        checkNullofMap(map);
+    }
+	
+    private void checkNullofMap(final Map<String, Object> map) {
```

Comment:
Also, it checks the size of map, so 'checkNullOrEmptyOfMap' is better for this method.